github/Repository.py

```diff
@@ -1264,6 +1264,7 @@ def create_git_tag_and_release(
         draft: bool = False,
         prerelease: bool = False,
         generate_release_notes: bool = False,
+        make_latest: str = "true",
```

Comment:
What is the current behaviour of Github with `make_latest` omitted?

Reply:
according to the official documentation, the default is "true"

Reply:
Why should make_latest be a string and not a bool?